pkg/log/log_file.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package log processes the klog flags, and enforces the maximum log file
+// size and maximum log file number limits.
+package log
+
+import (
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+)
+
+const (
+	logToStdErrFlag = "logtostderr"
+	logDirFlag      = "log_dir"
+	logFileFlag     = "log_file"
+	maxSizeFlag     = "log_file_max_size"
+	maxNumFlag      = "log_file_max_num"
+
+	// Check log file number every 10 mins.
+	logFileCheckInterval = time.Minute * 10
+	// Allowed maximum value for the maximum file size limit.
+	maxMaxSizeMB = 1024 * 100
```

Comment:
this was 1024*1024 previously.. decided to reduce?

Reply:
Yes. Do not really what is the appropriate number. klog default value is 18G.